mozilla-mobile/android-components/l10n.toml

```diff
@@ -36,9 +36,12 @@ locales = [
     "pl",
     "ru",
     "sk",
+    "su",
+    "sv-SE",
     "tr",
     "ta",
-    "sv-SE",
+    "uk",
```

Comment:
Do all projects (in this case Lockwise) require android-components? I thought that I would be able to determine that from the product's TOML file, but that's not the case.

Reply:
Yes. In a way, each android project is standalone, and one can just use the repository itself and gather an l10n status.

Related is the conversation in https://bugzilla.mozilla.org/show_bug.cgi?id=1605216.

Reply:
Answer and question seem disconnected. What I'm interested in is:
- Can a product in android-l10n not use android-component at all?
- If yes, how would I know?

If the answer is no, android-components should always be a superset of all individual locale lists.

Reply:
Firefox for Fire TV doesn't require a-c. There's no way to know from project config, the only way to know is to look at the build dependencies in the various Android projects.

Reply:
I wonder if it would be useful to have a comment in the TOML file, e.g. in Lockwise Android something that says "locales should be also enabled for android-components".